src/game/server/swarm/asw_button_area.cpp

```diff
@@ -615,6 +621,26 @@ bool CASW_Button_Area::KeyValue( const char *szKeyName, const char *szValue )
 		Q_strncpy( m_NoPowerMessage.GetForModify(), szValue, 255 );
 		return true;
 	}	
+	else if ( FStrEq( szKeyName, "usepanelmessage" ) )
```

Comment:
For safety (because many existing maps won't have these keyvalues set) we should do `m_*.GetForModify()[0] = '\0';` in the constructor.